.github/workflows/docker.yaml

```diff
@@ -1,6 +1,6 @@
 name: DockerImage 
 
-on: [workflow_call]
+on: [pull_request, push]
```

Comment:
`pull_request` is for testing this workflow. Should be removed before merging this PR.